bentoml/_internal/bento/gen.py

```diff
@@ -116,6 +121,48 @@ def get_templates_variables(
     }
 
 
+def process_extend_blocks(templates: nodes.Template) -> None | list[str]:
```

Comment:
We want to do this to catch use-case for custom docker templates where user can specify something like
```jinja
# my_template.j2
{% extends "miniconda_debian.j2" %}
```
while in `bentofile.yaml`:
```yaml
docker:
  distro: debian
  cuda_version: 11
  dockefile_template: ./my_template.j2
```